src/components/PageSections/Header/GovSite.js

```diff
@@ -1,17 +1,26 @@
 import React from 'react';
 
-import DomeSVG from 'components/SVGs/Dome';
-import { misc as i18n1, navigation as i18n2 } from 'js/i18n/definitions';
+import { misc as i18n1 } from 'js/i18n/definitions';
 import { injectIntl } from 'react-intl';
+import HowYouKnowMenu from 'components/PageSections/HowYouKnowMenu';
 
-const GovSite = ({ intl, toggleHowYouKnowMenu, menuIsOpen }) => (
-  <div className="coa-Header__gov-site">
-    <div className="container-fluid wrapper center-xs" onClick={toggleHowYouKnowMenu}>
+const GovSite = ({ intl, toggleHowYouKnowMenu, keyboardHowYouKnowMenu, menuIsOpen, refnode }) => (
+  <div
+  	className={menuIsOpen ? "coa-Header__gov-site-open" : "coa-Header__gov-site"}
+  	tabindex="0"
```

Comment:
Give us a little comment about why this was so important.

Reply:
added some comments!